README.rst

```diff
@@ -237,6 +238,51 @@ And then in your channels consumer, you can implement the handler:
     async def redis_disconnect(self, *args):
         # Handle disconnect
 
+
+
+``serializer_format``
+~~~~~~~~~~~~~~~~~~~~~~
+By default every message sent to redis is encoded using `msgpack <https://msgpack.org/>`_ (_currently ``msgpack`` is a mandatory dependency of this package, it may become optional in a future release_).
+It is also possible to switch to `JSON <http://www.json.org/>`_:
+
+.. code-block:: python
+
+    CHANNEL_LAYERS = {
+        "default": {
+            "BACKEND": "channels_redis.core.RedisChannelLayer",
+            "CONFIG": {
+                "hosts": ["redis://:password@127.0.0.1:6379/0"],
+                "serializer_format": "json",
+            },
+        },
+    }
+
+
+Custom serializer can be defined by:
+
+- extending ``channels_redis.serializers.BaseMessageSerializer``, implementing ``dumps`` and ``loads`` methods
```

Comment:
I think this needs to be updated to `as_bytes` and `from_bytes` right?

Reply:
You are right, I forgot to update the docs!